commune_app/models.py

```diff
@@ -1,3 +1,31 @@
-# from django.db import models
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import RegexValidator
 
-# Create your models here.
+# regex to filter valid phone numbers
+phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number format: '+999999999'")
+
+
+class User(AbstractUser):
+    id = models.IntegerField(unique=True, primary_key=True)
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    email = models.EmailField(max_length=250, unique=True)
+    phone_number = models.CharField(validators=[phone_regex], max_length=17, blank=True)
+
+
+class Expense(models.Model):
```

Comment:
```suggestion
class Expense(models.Model):
    id = models.IntegerField(unique=True, primary_key=True)
```
adding id so you can use it for the getter later